config/billing/generate_config.rb

```diff
@@ -18,14 +19,39 @@ def price(key)
   def generate(template)
     ERB.new(template).result(binding)
   end
+
+  def include(file)
+    fp = File.join(__dir__, file)
+    ERB.new(
+      File.read(File.expand_path(fp))
+    ).result(binding)
+  end
 end
 
 pricing_data = YAML.load_file("#{__dir__}/pricing_data.yml")
 template = File.read("#{__dir__}/config.json.erb")
 
 REGIONS.each do |region|
-  g = Generator.new(region, pricing_data)
-  File.open("#{__dir__}/output/#{region}.json", "w") do |f|
-    f.write g.generate(template)
+  begin
+    g = Generator.new(region, pricing_data)
+    json_content = g.generate(template)
+
+    JSON.parse json_content
+
+    File.open("#{__dir__}/output/#{region}.json", "w") do |f|
+      f.write json_content
```

Comment:
You could get a properly indented output (and a way nicer git diff in the output) if you did this:

```suggestion
      f.write JSON.pretty_generate(json, indent: "\t")
```

Reply:
Sorry - I missed saying that you'd also have to assign `json = JSON.parse json_content` on line 39.

Reply:
Done